assets/js/modules/ads/components/common/WooCommerceRedirectModal.js

```diff
@@ -68,6 +72,16 @@ export default function WooCommerceRedirectModal( {
 	const isGoogleForWooCommerceActive = useSelect( ( select ) =>
 		select( MODULES_ADS ).isGoogleForWooCommerceActivated()
 	);
+	const trackEventLabel = isGoogleForWooCommerceActive ? 'gfw' : 'wc';
+
+	useMount( () => {
+		trackEvent(
+			`${ viewContext }_pax_wc-redirect`,
+			'view_modal',
+			trackEventLabel
+		);
+	} );
```

Comment:
It looks like this component being viewed is primarily controlled by `dialogActive` – should we not track it based on this rather than simply by mount?

Reply:
Good point, I updated the implementation to use `useEffect` and based the tracking on `dialogActive`